docs/src/modules/components/Notifications.js

```diff
@@ -53,34 +59,34 @@ export default function Notifications() {
   const messages = useSelector((state) => state.notifications.messages);
   const lastSeen = useSelector((state) => state.notifications.lastSeen);
 
-  const messageList = (messages || [])
-    .filter((message) => {
-      if (
-        message.userLanguage &&
-        message.userLanguage !== userLanguage &&
-        message.userLanguage !== navigator.language.substring(0, 2)
-      ) {
-        return false;
-      }
-      return true;
-    })
-    .reverse();
-
-  const unseenNotificationsCount = messageList.reduce(
-    (count, message) => (message.id > lastSeen ? count + 1 : count),
-    0,
-  );
+  const messageList = messages
+    ? messages
+        .filter((message) => {
+          if (
+            message.userLanguage &&
+            message.userLanguage !== userLanguage &&
+            message.userLanguage !== navigator.language.substring(0, 2)
+          ) {
+            return false;
+          }
+          return true;
+        })
+        .reverse()
+    : null;
 
   const handleToggle = () => {
     setOpen((prevOpen) => !prevOpen);
     setTooltipOpen(false);
-    dispatch({
-      type: ACTION_TYPES.NOTIFICATIONS_CHANGE,
-      payload: {
-        lastSeen: messageList[0].id,
-      },
-    });
-    document.cookie = `lastSeenNotification=${messageList[0].id};path=/;max-age=31536000`;
+
+    if (messageList) {
```

Comment:
All though it shouldn't happen if this is an empty array below will still throw.

```suggestion
    if (messageList && messageList.length > 0) {
```